kartothek/io/testing/read.py

```diff
@@ -267,7 +267,6 @@ def test_read_dataset_as_dataframes_predicate(
         [[("P", "in", [2]), ("TARGET", "==", 2)]],
         [[("P", "!=", 1), ("L", "==", 2)]],
         [[("P", "!=", 1), ("L", "in", [2])]],
-        [[("P", ">", 2)], [("TARGET", ">=", 2)]],
```

Comment:
Please review this thoroughly. It seems to me that we had a bug in the partition pruning before this refactoring. The previously tested behaviour seems wrong to me.